src/app/shared/entry-wizard/entry-wizard.component.ts

```diff
@@ -25,7 +26,16 @@ export class EntryWizardComponent implements OnInit {
     this.isLoading$ = this.entryWizardQuery.selectLoading();
     this.gitRegistries$ = this.entryWizardQuery.selectGitRegistries$;
     this.gitOrganizations$ = this.entryWizardQuery.selectGitOrganizations$;
-    this.gitRepositories$ = this.entryWizardQuery.selectGitRepositories$;
+    this.gitRepositories$ = this.entryWizardQuery.selectGitRepositories$.pipe(
+      map((repositories) => {
+        if (repositories !== undefined) {
+          repositories.sort((a, b) => a.repositoryName.localeCompare(b.repositoryName));
+          return repositories;
+        } else {
+          return null;
```

Comment:
Changed to return null instead of an empty list because returning an empty list resulted in the repository field appearing before an organization was selected.